backend/src/main/java/team/teamby/teambyteam/feed/application/FeedThreadService.java

```diff
@@ -37,4 +51,40 @@ public Long write(
         final Feed savedFeed = feedRepository.save(feed);
         return savedFeed.getId();
     }
+
+    @Transactional(readOnly = true)
+    public FeedsResponse firstRead(final Long teamPlaceId, final Integer size) {
+        final Pageable pageable = getPageableInitSize(size);
+        final List<Feed> list = feedRepository.findByTeamPlaceId(teamPlaceId, pageable);
+        final List<FeedResponse> feedResponses = mapFeedResponses(list);
+
+        return FeedsResponse.of(feedResponses);
+    }
+
+    @Transactional(readOnly = true)
+    public FeedsResponse reRead(final Long teamPlaceId, final Long threadId, final Integer size) {
+        final Pageable pageable = getPageableInitSize(size);
+        final List<Feed> list = feedRepository.findByTeamPlaceIdAndIdLessThan(teamPlaceId, threadId, pageable);
+        final List<FeedResponse> feedResponses = mapFeedResponses(list);
+
+        return FeedsResponse.of(feedResponses);
+    }
+
+    private Pageable getPageableInitSize(final Integer size) {
+        return PageRequest.of(FIRST_PAGE, size, SORT_DIRECTION, SORT_CRITERIA);
+    }
+
+    private List<FeedResponse> mapFeedResponses(final List<Feed> feeds) {
+        return feeds.stream().map(feed -> {
+            if (FeedType.THREAD == feed.getType()) {
+                final Optional<Member> member = memberRepository.findById(feed.getAuthorId());
+                final Member member1 = member.orElseThrow(MemberException.MemberNotFoundException::new);
+                return FeedResponse.from(feed, member1.getName().getValue(), member1.getProfileImageUrl().getValue());
+            }
+            if (FeedType.SCHEDULE_NOTIFICATION == feed.getType()) {
+                return FeedResponse.from(feed, "schedule", "");
+            }
+            throw new IllegalArgumentException("지원하지 않는 타입입니다.");
+        }).toList();
+    }
```

Comment:
```suggestion
    private List<FeedResponse> mapFeedResponses(final List<Feed> feeds) {
        return feeds.stream()
                .map(this::mapToResponse)
                .toList();
    }

    private FeedResponse mapToResponse(final Feed feed) {
        if (FeedType.THREAD == feed.getType()) {
            final Optional<Member> member = memberRepository.findById(feed.getAuthorId());
            final Member member1 = member.orElseThrow(MemberException.MemberNotFoundException::new);
            return FeedResponse.from(feed, member1.getName().getValue(), member1.getProfileImageUrl().getValue());
        }
        if (FeedType.SCHEDULE_NOTIFICATION == feed.getType()) {
            return FeedResponse.from(feed, "schedule", "");
        }
        throw new IllegalArgumentException("지원하지 않는 타입입니다.");
    }
```

다음과 같이 메서드 분리해서 리팩터링 할 수 있을 것 같습니다.
이런식으로 해두면 후에 enum타입을 활용해서 response를 만드는 방식등에 패턴을 적용하거나 할 때도 `mapToResponse`로 빼둔 메서드만 리팩터링 하는 등의 방법으로 쉽게 할 수 있고 가독성도 좋아질 것 같습니다.

Reply:
적용 완료 했습니다.